client/dashboard/top-selling-products/__mocks__/mock-data.js

```diff
@@ -9,7 +9,8 @@ And as such will require data layer logic for products to fully build the table
     "product_id": 20,
     "items_sold": 100,
     "gross_revenue": 999.99,
-    "orders_count": 54,
+		"orders_count": 54,
+		"name": 'Product name',
```

Comment:
Not related to this PR, but I just saw we are mixing spaces and tabs in the mock comment. :slightly_frowning_face: